.pre-commit-config.yaml

```diff
@@ -41,6 +41,7 @@ repos:
         - -d too-few-public-methods
         - -d too-many-arguments
         - -d useless-object-inheritance  # necessary for Python 2 compatibility
+        - -d too-many-arguments # TODO stop ignoring this and fix the code
```

Comment:
The `pre-commit-config` changes remain from the previous version of the PR - I can revert them. But I think this file should also ignore `too-many-arguments`